modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Tests/TestData/Sources/MoreExportedFields.cs

```diff
@@ -0,0 +1,8 @@
+using Godot;
+using System;
+
+public partial class ExportedFields : GodotObject
+{
+    // Note we use Array and not System.Array. This tests the generated namespace qualification.
+    [Export] private Int64[] field_empty_Int64Array = Array.Empty<Int64>();
```

Comment:
I really don't know TBH. All the files under `TestData/` come from the `Godot.SourceGenerators.Sample` and haven't been modified. If we check this somewhere else, it might be good to simplify this test.

Reply:
Yeah, it makes sense to keep the tests unmodified for the initial implementation, they can always be simplified/improved in follow-up PRs.

Looking at the original, there's a comment that's missing here.

https://github.com/godotengine/godot/blob/04bed2bbd50c7f225747e608634eeb7eb7c7214b/modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Sample/MoreExportedFields.cs#L13

So it seems the intention is to test exported fields across `partial` declarations. So I guess it makes more sense now, I don't think we have other tests for this (the closest is probably `Foo.cs` and `Bar.cs` but those don't expose fields/properties.